### DISCUSSION TITLE
Column types

### DISCUSSION THREAD

**Discussion by shivarajmulimani** [cozodb/cozo#133] [Q&A]
I am trying to create a table like below
`{:create country {
    code: String,
    =>
    desc: Json
}}`

As per documentation I see column type can be Json - [https://docs.cozodb.org/en/latest/datatypes.html#column-types](https://docs.cozodb.org/en/latest/datatypes.html#column-types)
But I am getting below error

`parser::pest

  × The query parser has encountered unexpected input / end of input at 55..55
   ╭─[3:1]
 3 │     =>
 4 │     desc: Json
   ·           ▲
 5 │ }}
   ╰────`

**Comment by zh217** [comment]
You must be running an old version. Please upgrade to the latest release.
